.github/workflows/codesee-arch-diagram.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow was added by CodeSee. Learn more at https://codesee.io/
```

Comment:
This file should be removed from this PR.
It is unrelated to the purpose of the PR.